arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -944,13 +979,25 @@ Result RocksDBCollection::insert(arangodb::transaction::Methods* trx,
     guard.finish(hasPerformedIntermediateCommit);
   }
 
+  _statistics._rocksdb_insert_msec.count(duration<float,std::micro>(clock::now() - start).count());
   return res;
 }
 
 Result RocksDBCollection::update(transaction::Methods* trx,
                                  velocypack::Slice newSlice,
                                  ManagedDocumentResult& resultMdr, OperationOptions& options,
                                  ManagedDocumentResult& previousMdr) {
+
+  if (options.isSynchronousReplicationFrom.empty()) {
+    ++_statistics._numWrite;
```

Comment:
We now have seperate metrics for insert, update and remove when it comes to timers.
But we don't have seperate metrics for insert, update and remove when it comes to counters.
Should we also split the counters in the same way as the timers?

Reply:
that is correct. this was also intended. write / read / replicate are one class of counters. 
the much more specific insert, ... operations are recorded by the histograms, that inherently collect counters as a sum of all records.